### PR TITLE
[IN-2514] Hardcode go version

### PR DESCRIPTION
**Pull request by fehersanyi-bitrise**
### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [ ] The tool I added is stable, and __does not require frequent updates__. _Preinstalled tools on the LTS stacks
  are not updated, so if the tool requires frequent updates you should handle the installation on-demand (see: [Install Any Additional Tool - DevCenter](https://bitrise-io.github.io/devcenter/tips-and-tricks/install-additional-tools/) for more information)_
- [ ] I updated the corresponding tests if there were any.
- [ ] I added a version report line to [`system_report.sh`](/system_report.sh) for the new tool(s) I added
- [ ] I have added an entry to the CHANGELOG.md with a revisionID (the current date and the number of the change e.g.: 2019_10_11_1) and a brief description of the actual change
- [ ] I have updated the revisionID in [bitrise_profile](roles/profiles/files/bitrise_profile) **BITRISE_OSX_STACK_REV_ID** with the one I just created in the CHANGELOG.md

### CODE REVIEW

**Review by tamas-bitrise — DISMISSED** [review]
Can the go version come from a variable in the test_go.py so you don't have to update it every time the version changes?

**Review by nagypeterjob — APPROVED** [review]
lgtm

File: roles/go/tasks/main.yml [hunk 1]
```diff
@@ -1,11 +1,38 @@
...
+  when: go_installation.rc != 0 or go_installation.stdout != "{{ go_version }}"
+
+- name: install Go
+  file:
+    src: "{{ ansible_env.HOME }}/go/bin/go"
+    dest: /usr/local/bin/go
+    owner: "{{ param_user }}"
+    mode: "0755"
+    state: link
+  become: true
```
Comment (peterheja-bitrise) [inline roles/go/tasks/main.yml:34]: Delete this.

File: roles/go/tasks/main.yml [hunk 2]
```diff
@@ -1,11 +1,38 @@
...
+  file:
+    path: "{{ ansible_env.HOME }}/go"
+    state: absent
+  when: go_installation.stdout != "{{ go_version }}"
+
+- name: Download Go
+  get_url:
+    url: https://golang.org/dl/{{ go_version }}.{{ ansible_system | lower }}-amd64.tar.gz
+    dest: /tmp/go{{ go_version }}.zip
+  when: go_installation.rc != 0 or go_installation.stdout != "{{ go_version }}"
```
Comment (bitrise-tom) [inline roles/go/tasks/main.yml:18]: these look nicer and more readable on mutliline:
something i learned recently
```
when: >
  go_installation.rc != 0 or
  go_installation.stdout != "{{ go_version }}"
```

File: roles/go/tasks/main.yml (hunk 1 above)
Comment (fehersanyi-bitrise) [reply roles/go/tasks/main.yml:34]: but you know that /usr/local/bin can not be written as a mortal user? if I remove it the task fails 🧐

File: roles/go/tasks/main.yml (hunk 2 above)
Comment (fehersanyi-bitrise) [reply roles/go/tasks/main.yml:18]: looks nice, I'll do this one

Comment (peterheja-bitrise) [reply roles/go/tasks/main.yml]:
```
when: >
  go_installation.rc != 0
  or go_installation.stdout != "{{ go_version }}"
```

Comment (fehersanyi-bitrise) [reply roles/go/tasks/main.yml]: I agree with @bitrise-tom on this one :( I like the first one better

**Review by nagypeterjob — APPROVED** [review]
lgtm

File: roles/go/tasks/main.yml (hunk 2 above)
Comment (peterheja-bitrise) [reply roles/go/tasks/main.yml]: We use both 
https://github.com/bitrise-io/osx-box-bootstrap/blob/master/roles/ruby-gems/tasks/main.yml#L34
https://github.com/bitrise-io/osx-box-bootstrap/blob/master/roles/ruby-gems/tasks/main.yml#L45

The operator in the new line can be a bit better to read.
(And @bitrise-tom convinced me about that one earlier :D)

I'm okay with either, but still think the operator on new-line is cleaner a bit.

**Review by peterheja-bitrise — APPROVED** [review]
LGTM

**Pull request merged**